osdep/meson.build

```diff
@@ -33,7 +33,8 @@ swift_flags += extra_flags
 swift_compile = [swift_prog, swift_flags, '-module-name', 'macOS_swift',
                  '-emit-module-path', '@OUTPUT0@', '-import-objc-header', bridge,
                  '-emit-objc-header-path', '@OUTPUT1@', '-o', '@OUTPUT2@',
-                 '@INPUT@', '-I.', '-I' + source_root]
+                 '@INPUT@', '-I.', '-I' + source_root,
+                 '-I' + libplacebo.get_variable('includedir', default_value: '.')]
```

Comment:
Not sure the default value is right? I don't think either of us can really test but I think it should be the absolute path to the libplacebo subproject include directory since that's the only case where the pkgconfig variable would fail.

Reply:
Default value is only a placeholder to make command correct. It duplicates another `-I.` But sure, let's make as it should be.